src/audio/codec_adapter/CMakeLists.txt

```diff
@@ -13,4 +13,5 @@ endif()
 if(CONFIG_WAVES_CODEC)
 add_local_sources(sof codec/waves.c)
 sof_add_static_library(MaxxChrome ${CMAKE_CURRENT_LIST_DIR}/lib/release/libMaxxChrome.a)
+target_include_directories(sof PUBLIC ${CMAKE_CURRENT_LIST_DIR}/../../include/sof/audio/)
```

Comment:
@stolx thanks for moving the header file location. It looks good now. 

I have one question though. Do you really need to add the line above? I would have expected to just work. If it is really needed that fine by me. The change looks good and useful!

Reply:
agreed, if your stuff is in include you shouldnt need this

Reply:
fixed include dir to be `${CMAKE_CURRENT_LIST_DIR}/../../include/sof/audio/codec_adapter/codec/`
we need it to keep MaxxEffect folder on includes path

Reply:
@stolx OK, I got your point. It works for me. Another approach would have been to use
```
include "codec_adapter/codec/MaxxEffect/MaxxEffect.h"
```

Reply:
unfortunately, it will not work due to how MaxxEffect API is designed.
For example, take a look at `MaxxEffect_Revision.h`, it contains the following lines
`#include "MaxxEffect/MaxxEffect.h"
#include "MaxxEffect/MaxxStatus.h"`
So API headers themself expect MaxxEffect folder to be on the include path.
And since API headers are provided "as-is", the only way is to add the MaxxEffect folder to includes

Reply:
@stolx thanks for your explanation! It makes sense.